terrat_runner/github_actions/run_time.py

```diff
@@ -123,3 +123,14 @@ def group_output(self, title, output):
 
     def update_workflow_steps(self, run_type, steps):
         return [{'type': 'update_terrateam_github_token'}] + steps
+
+    def is_command(self, str):
+        return str.startswith('::add-mask::')
+
+    def extract_secrets(self, str):
+        secrets = []
+        for s in str.splitlines():
+            if s.startswith('::add-mask::'):
+                secrets.append(s[len('::add-mask::'):])
+
+        return secrets
```

Comment:
Some suggestion to avoid copy and pasted code on `cmd.py` 

```suggestion
        return secrets
        
    def handle_stream(stream, working_dir, stream_name):
        stream.write(line)
        if self.is_command(line):
            sys.stdout.write(line)
        elif config.get('log_output', True):
            sys.stderr.write(f"cwd={working_dir}: {stream_name}: {line}")
```